spec/app_room_view_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "spec_helper"
+
+RSpec.describe "Room view", type: :feature do
+  before do
+    Capybara.app = Sinatra::Application
+    stub_const("CACHE_EXPIRY_TIMEOUT", 0) # Overriding this constant so the response won't stay cached between examples
+  end
+
+  context 'get "/room/:slug"' do
+    context "when there are no meetings for the day" do
+      it "shows the room is empty until the next day" do
+        stub_gcal_request(gcal_fake_response([]))
+
+        visit "/room/hoxton_wellbeing"
+
+        expect(page).to have_selector(".event.empty")
```

Comment:
This is a great example of a test which is doing one thing only - it won't be affected if we rename the room, restyle the page etc.

Reply:
👍